README.md

```diff
@@ -16,7 +16,7 @@ Add this dependency to your project's POM:
 <dependency>
   <groupId>com.stripe</groupId>
   <artifactId>stripe-java</artifactId>
-  <version>7.29.0</version>
+  <version>8.0.0</version>
```

Comment:
You are not supposed to change this yourself. The release script does this for you.